client/my-sites/google-my-business/stats/index.js

```diff
@@ -220,7 +221,22 @@ class GoogleMyBusinessStats extends Component {
 				<QueryKeyringConnections forceRefresh />
 				<QueryKeyringServices />
 
-				{ ! isLocationVerified && (
+				{ ! locationData && (
+					<Notice
+						status="is-error"
+						text={ translate(
+							'There is an error with your Google My Business Connection. ' +
+								'Please {{a}}contact support{{/a}}',
```

Comment:
We're missing a trailing dot here. I'd like to propose the following since we don't use the term _connection_ anywhere else, and users may be confused:

> There was an error connecting to your Google My Business account. Please contact support.

Reply:
yeah, I was trying to differentiate from the actual account they manage through Google, but since we are sending them to support regardless so `account` is probably good!